understanding/22/target-size-minimum.html

```diff
@@ -31,7 +31,7 @@ <h3>Exceptions</h3>
             <ul>
                 <li><strong>Spacing:</strong> Undersized targets (those less than 24 by 24 CSS pixels) are positioned so that if a 24 CSS pixel diameter circle is centered on the <a>bounding box</a> of each, the circles do not intersect another target or the circle for another undersized target.</li>
                 <li><strong>Equivalent:</strong> In cases where a target does not have a size equivalent to 24 by 24 CSS pixels, but there is another control that can achieve the underlying function that <em>does</em> meet the minimum size, the target can be excepted based on the "Equivalent" exception.</li>
-                <li><strong>Inline:</strong> The Success Criterion does not apply to inline targets in sentences, or where the size of the target is constrained by the line-height of non-target text. This exception is allowed because text reflow based on viewport size makes it impossible for authors to anticipate where links may be positioned relative to one another. When multiple links are embedded in blocks of texts in smaller text sizes, guaranteeing that undersized links in adjacent lines of text fulfill the spacing exception (their 24 CSS pixel diameter circle don't intersect any other links or their circles) would require a large line height which can be undesirable in many design contexts.</li>
+                <li><strong>Inline:</strong> The Success Criterion does not apply to inline targets in sentences, or where the size of the target is constrained by the line-height of non-target text. This exception is allowed because text reflow based on viewport size makes it impossible for authors to anticipate where links may be positioned relative to one another. When multiple links are embedded in blocks of smaller text, guaranteeing that undersized links in adjacent lines fulfil the spacing exception could result in an undesirable design in many contexts; the line height would need to be set to a value that could affect affect readability.</li>
```

Comment:
```suggestion
                <li><strong>Inline:</strong> The Success Criterion does not apply to inline targets in sentences, or where the size of the target is constrained by the line-height of non-target text. This exception is allowed because text reflow based on viewport size makes it impossible for authors to anticipate where links may be positioned relative to one another. When multiple links are embedded in blocks of smaller text, guaranteeing that undersized links in adjacent lines fulfill the spacing exception could result in an undesirable design in many contexts; the line height would need to be set to a value that could affect affect readability.</li>
``` 

would use “fulfill” because US spelling is more consistent with everything else in W3C

Reply:
```suggestion
                <li><strong>Inline:</strong> The Success Criterion does not apply to inline targets in sentences, or where the size of the target is constrained by the line-height of non-target text. This exception is allowed because text reflow based on viewport size makes it impossible for authors to anticipate where links may be positioned relative to one another. When multiple links are embedded in blocks of smaller text, guaranteeing that undersized links in adjacent lines fulfill the spacing exception could result in an undesirable design in many contexts; the line height would need to be set to a value that could affect affect readability.</li>
```

would use “fulfill” because US spelling is more consistent with everything else in W3C